src/Batch/FileConventions/Src/AzureBatchFileConventions/Properties/AssemblyInfo.cs

```diff
@@ -19,8 +19,8 @@
 [assembly: AssemblyTitle("Azure Batch File Conventions")]
 [assembly: AssemblyDescription("A convention-based library for saving and retrieving Azure Batch task output files.")]
 
-[assembly: AssemblyVersion("1.0.0.0")] 
-[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyVersion("1.0.1.0")] 
```

Comment:
Can you explain this a bit more @markcowl? Why should we not change the assembly version?